ReManga/Api/ReManga/ReMangaApi.swift

```diff
@@ -98,19 +98,22 @@ class ReMangaApi: ApiProtocol {
         }
 
         let url = "https://api.remanga.org/api/search/catalog/?count=30&ordering=-\(sorting.id)&page=\(page)\(tags)"
-        let model: ReMangaApiMangaCatalogResult = try await performRequest(makeRequest(url))
-
-        var bookmarks: [ApiMangaBookmarkModel] = []
-        do { bookmarks = try await fetchBookmarkTypes() }
-        catch { print(error) }
+        do {
+            let model: ReMangaApiMangaCatalogResult = try await performRequest(makeRequest(url))
```

Comment:
This line was failed. I've added in under `do catch` block to handle this in a future

Reply:
Could you, please, remove `do catch` from here? Networking layer is not the place for such logic, it should throw Error as it is, and it's responsibility of ViewModel to handle it properly